modules/ROOT/pages/afm-edit-test.adoc

```diff
@@ -0,0 +1,23 @@
+= Create a New Version of an Existing Test
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+You cannot edit a test that exists for a monitor, but you can create a new version of one. In the new version, you can change, remove, or add assertions, headers, endpoints, and reporters.
+
+You can create a new version of a test only if the test has not been customized to include features that the *Create a Monitor from Scratch* page does not support. (It is possible to customize a test by downloading it, editing it manually, and then uploading it.) Tests created manually and then uploaded also are not editable if they contain features that the *Create a Monitor from Scratch* page does not support.
```

Comment:
ditto re. "create a new version"

Reply:
This paragraph is confusing to me. Is it more advantageous to "create a new version" "onsite" versus manually downloading, editing, and uploading? If neither can contain features not supported by the page, why state this twice? Do we really want to say that you cannot edit a test that is being used, but you can "create a new version," either on-premises or manually, by copying and editing or by downloading, editing, and uploading, as long as the new version does not contain features....